src/postscribe.js

```diff
@@ -1,4 +1,4 @@
-import WriteStream from './WriteStream';
+import WriteStream from './write-stream';
```

Comment:
Why did you rename this?  From what I've seen, it's somewhat customary to name the file after class where the default export is a class.